implementations/rust/ockam/ockam_command/src/node/mod.rs

```diff
@@ -123,13 +123,19 @@ pub fn spawn_default_node(node_name: &str) -> String {
     create_command.run(quiet_opts);
 
     if let Ok(mut logs) = PARSER_LOGS.lock() {
-        logs.push(fmt_log!("No default node was found."));
         logs.push(fmt_log!(
-            "Created default node, {}",
+            "There is no node, on this machine, marked as your default."
```

Comment:
Typo in this comma here? `no node, on this`

Reply:
I believe the comma is correct here?

"There is no node marked as your default."

", on this machine,"